data/src/main/kotlin/info/dvkr/screenstream/data/httpserver/HttpServerFiles.kt

```diff
@@ -75,6 +75,7 @@ class HttpServerFiles(context: Context, private val settingsReadOnly: SettingsRe
         const val TOGGLE_STREAM_ADDRESS = "toggle-stream"
         const val CHANGE_IMAGE_SIZE_ADDRESS = "image-size"
         const val CHANGE_IMAGE_COMPRESSION_ADDRESS = "image-compression"
+        const val GET_JSON_ADDRESS = "/get-json"
```

Comment:
We might need to make a change here. The way pin security is set up, it garbles the address for certain endpoints like the image stream and picture quality. We will want to be able to garble the json address as well. 

Let me look into this a little more.

Reply:
This endpoint calls:

```
response.sendJSON(httpServerFiles.configureCoPlayJSON(
                        httpServerFiles.configureStreamAddress(),
                        httpServerFiles.configureAppActionAddress(),
                        httpServerFiles.configureSystemActionAddress()
                ))
```

And the logic for PIN garbling is contained within those 3 configure functions, so should still work